config/settings.yml

```diff
@@ -13,8 +13,6 @@ zotero_api:
   solr_document_field: zotero_bibliography_tsi
```

Comment:
these `zotero_api` settings can also be deleted

Reply:
Good catch. Pushed